dev-util/bencode/bencode_tools-20110315.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="C and Python tools for manipulating bencoded data"
+DESCRIPTION="bencode-tools is a collection of tools for manipulating \
+bencoded data. See http://en.wikipedia.org/wiki/Bencode for information \
+on bencode format.
+
+This was formerly named typevalidator. With the introduction of \
+libbencodetools the project is named bencode-tools."
+HOMEPAGE="http://zakalwe.fi/~shd/foss/bencode-tools/"
+COPYRIGHT="2002-2005 Bram Cohen and Ross Cohen"
+LICENSE="Bencode"
```

Comment:
OK, so it is indeed the BSD 3-clause, but there is also another LICENSE file that says everything but the python script is public domain. So this should have 2 entries: Public Domain, and BSD 3-clause, and you should delete the custom license.